docs/contentlayer.config.ts

```diff
@@ -18,6 +18,17 @@ const Component = defineDocumentType(() => ({
             resolve: (doc) => doc._raw.sourceFileDir.toLocaleLowerCase(),
         },
         route: {
+            type: 'string',
+            resolve: (doc) => {
+                const pathArray = doc._raw.flattenedPath.toLocaleLowerCase().split('/');
+                if (pathArray.at(-1) === pathArray.at(-2)) {
+                    return pathArray.at(-2);
+                } else {
+                    return pathArray.slice(-2).join('/');
+                }
+            },
+        },
+        parentFolder: {
             type: 'string',
             resolve: (doc) => doc._raw.sourceFileDir.toLocaleLowerCase().split('/').at(-1),
         },
```

Comment:
`parentFolder` should be `.at(-2)`

`'/fondue/tag/content'.split('/')`:
- `.at(-1)` is `content`
- `.at(-2)` is `tag`